README.md

```diff
@@ -675,7 +675,7 @@ Other Style Guides
 
     ```javascript
     // immediately-invoked function expression (IIFE)
-    (function () {
+    (function() {
```

Comment:
I'm not sure what rule you mean; there must *always* be a single space after the `function` keyword, no exceptions.